qt/aqt/preferences.py

```diff
@@ -404,15 +401,18 @@ def update_video_driver(self) -> None:
 
 
 def video_driver_name_for_platform(driver: VideoDriver) -> str:
-    if driver == VideoDriver.ANGLE:
-        return tr.preferences_video_driver_angle()
-    elif driver == VideoDriver.Software:
-        if is_mac:
-            return tr.preferences_video_driver_software_mac()
-        else:
-            return tr.preferences_video_driver_software_other()
-    else:
-        if is_mac:
-            return tr.preferences_video_driver_opengl_mac()
-        else:
-            return tr.preferences_video_driver_opengl_other()
+    if qtmajor < 6:
+        if driver == VideoDriver.ANGLE:
+            return tr.preferences_video_driver_angle()
+        elif driver == VideoDriver.Software:
+            if is_mac:
+                return tr.preferences_video_driver_software_mac()
+            else:
+                return tr.preferences_video_driver_software_other()
+        elif driver == VideoDriver.OpenGL:
+            if is_mac:
+                return tr.preferences_video_driver_opengl_mac()
+            else:
+                return tr.preferences_video_driver_opengl_other()
+
+    return driver.name
```

Comment:
@dae should we add strings for the new defaults (with a "recommended" label)?

Reply:
What if we just used English strings like "Metal", and added a fluent key so we can add "default" to whatever is the platform default - e.g. "Metal (default)"? "default" might be a better choice than "recommended", because what's most suitable depends on the user's system.